static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issues.tsx

```diff
@@ -126,33 +128,35 @@ function IssueListHeader({node}: {node: TraceTreeNode<TraceTree.NodeValue>}) {
   return (
     <StyledPanelHeader disablePadding>
       <IssueHeading>
-        {errors.length > 0 && performance_issues.length === 0
-          ? tct('[count] [text]', {
-              count: errors.length,
-              text: tn('Error', 'Errors', errors.length),
-            })
-          : performance_issues.length > 0 && errors.length === 0
+        {errors.length + performance_issues.length > MAX_DISPLAYED_ISSUES_COUNT
+          ? t(`%s+  issues`, MAX_DISPLAYED_ISSUES_COUNT)
+          : errors.length > 0 && performance_issues.length === 0
             ? tct('[count] [text]', {
-                count: performance_issues.length,
-                text: tn(
-                  'Performance issue',
-                  'Performance Issues',
-                  performance_issues.length
-                ),
+                count: errors.length,
+                text: tn('Error', 'Errors', errors.length),
               })
-            : tct(
-                '[errors] [errorsText] and [performance_issues] [performanceIssuesText]',
-                {
-                  errors: errors.length,
-                  performance_issues: performance_issues.length,
-                  errorsText: tn('Error', 'Errors', errors.length),
-                  performanceIssuesText: tn(
-                    'performance issue',
-                    'performance issues',
+            : performance_issues.length > 0 && errors.length === 0
+              ? tct('[count] [text]', {
+                  count: performance_issues.length,
+                  text: tn(
+                    'Performance issue',
+                    'Performance Issues',
```

Comment:
Will this ever run? Because line 133 looks like it already checks the same thing

Reply:
Yeah one checks for errors no perf issues and the other checks for perf issues and no errors

Reply:
OH I misread lol